resources/views/meetings/edit-times.blade.php

```diff
@@ -0,0 +1,21 @@
+@if ($user->id == $meeting->user_id)
```

Comment:
Needs also a check if user is authenticated, because if not throws an exception:
Attempt to read property "id" on null